spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -277,6 +277,7 @@ def project_path_update(self, project_path, update_kind):
                     )
                     self.main.projects.stop_workspace_services()
                     self.main.editor.stop_completion_services(language)
+                    self.main.outlineexplorer.stop_symbol_services(lamguage)
```

Comment:
I think that here is a typo `lamguage` -> `language`

Reply:
Thanks @steff456, great eye!